src/main/java/redis/clients/jedis/BinaryJedisCluster.java

```diff
@@ -0,0 +1,1260 @@
+      }
+    }.runBinary(key);
+  }
+
+  @Override
+  public Long zremrangeByScore(final byte[] key, final double start, final double end) {
+    return new JedisClusterCommand<Long>(connectionHandler, timeout, maxRedirections) {
+      @Override
+      public Long execute(Jedis connection) {
+        return connection.zremrangeByScore(key, start, end);
+      }
+    }.runBinary(key);
+  }
+
+  @Override
+  public Long zremrangeByScore(final byte[] key, final byte[] start, final byte[] end) {
+    return new JedisClusterCommand<Long>(connectionHandler, timeout, maxRedirections) {
+      @Override
+      public Long execute(Jedis connection) {
+        return connection.zremrangeByScore(key, start, end);
+      }
+    }.runBinary(key);
+  }
+
+  @Override
+  public Long linsert(final byte[] key, final Client.LIST_POSITION where, final byte[] pivot,
+      final byte[] value) {
+    return new JedisClusterCommand<Long>(connectionHandler, timeout, maxRedirections) {
+      @Override
+      public Long execute(Jedis connection) {
+        return connection.linsert(key, where, pivot, value);
+      }
+    }.runBinary(key);
+  }
+
+  @Override
+  public Long lpushx(final byte[] key, final byte[]... arg) {
+    return new JedisClusterCommand<Long>(connectionHandler, timeout, maxRedirections) {
+      @Override
+      public Long execute(Jedis connection) {
+        return connection.lpushx(key, arg);
+      }
+    }.runBinary(key);
+  }
+
+  @Override
+  public Long rpushx(final byte[] key, final byte[]... arg) {
+    return new JedisClusterCommand<Long>(connectionHandler, timeout, maxRedirections) {
+      @Override
+      public Long execute(Jedis connection) {
+        return connection.rpushx(key, arg);
+      }
+    }.runBinary(key);
+  }
+
+  @Override
+  public List<byte[]> blpop(final byte[] arg) {
+    return new JedisClusterCommand<List<byte[]>>(connectionHandler, timeout, maxRedirections) {
+      @Override
+      public List<byte[]> execute(Jedis connection) {
+        return connection.blpop(arg);
+      }
+    }.runBinary(arg);
+  }
+
+  @Override
+  public List<byte[]> brpop(final byte[] arg) {
+    return new JedisClusterCommand<List<byte[]>>(connectionHandler, timeout, maxRedirections) {
+      @Override
+      public List<byte[]> execute(Jedis connection) {
+        return connection.brpop(arg);
+      }
+    }.runBinary(arg);
+  }
+
+  @Override
+  public Long del(final byte[] key) {
+    return new JedisClusterCommand<Long>(connectionHandler, timeout, maxRedirections) {
+      @Override
+      public Long execute(Jedis connection) {
+        return connection.del(key);
+      }
+    }.runBinary(key);
+  }
+
+  @Override
+  public byte[] echo(final byte[] arg) {
+    return new JedisClusterCommand<byte[]>(connectionHandler, timeout, maxRedirections) {
+      @Override
+      public byte[] execute(Jedis connection) {
+        return connection.echo(arg);
+      }
+    }.runBinary(null);
```

Comment:
@christophstrobl 
Thanks for finding missed spot! I'll fix it.

Btw, I think echo should belong to BasicCommand, as all commands with no key belong to BasicCommand, and echo doesn't have actual operation about key space.

@xetorthio @marcosnils @nykolaslima What do you think?